<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabCont">
       <attribute name="title">
        <string>Containers</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="2">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContentsCont">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1035</width>
             <height>557</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayoutCont"/>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCommandLinkButton" name="addBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ADD</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCommandLinkButton" name="delAllContButton">
          <property name="text">
           <string>Delete All</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabNet">
       <attribute name="title">
        <string>Networks</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="3">
         <widget class="QScrollArea" name="scrollAreaNet">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContentsNet">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1035</width>
             <height>557</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayoutNet"/>
          </widget>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCommandLinkButton" name="delAllNetBtn">
          <property name="text">
           <string>Prune</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCommandLinkButton" name="addBtnNet">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ADD</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
